pages/Embed Media - Audio, Photos, Videos.md

```diff
@@ -1,127 +1,135 @@
-		- [:audio {:controls true :src "https://www.kozco.com/tech/piano2-CoolEdit.mp3"}]
-	- ### Embedding an audio file on your PC (using [[Hiccup]])
-		- To preview audio saved on your computer, type or copy-paste the following Hiccup ClojureScript, replacing the source `src` in "quotation marks" with the filepath and file name:
+- Logseq lets you embed, preview, and play back a variety of media in a number of ways.
+-
+- For the most part, there are three ways that you can embed media (with `examples`):
+	- 1. Using Markdown: `![](Link-To-File)`
+	- 2. Using HTML elements: `<element src="Link-To-File"></element>`
+	- 3. Using [[Hiccup]] ClojureScript: `[:element :src "Link-To-File"}]`
+-
+- The source link to file will vary depending on whether you're embedding a local asset (a file on your computer) or an asset stored on an online server.
+	- **For online assets:** Copy the URL of the media, making sure that the file name and extension is included.
+		- ex: `https://www.mysite.com/file.mp3`
+	- **For local assets:** Copy your file path (including the file name and extension). You may also need to add `file:///` at the beginning.
+		- ex: `C:/Users/user/Videos/file.mp4`
+-
+- Here's a detailed breakdown of the variety of ways you can embed audio, video, and images to your Logseq pages:
+-
+- # Markdown Method
+	- The Markdown method is the easiest to implement. It involves embedding media in the following way:
+	- ```markdown
+	  - ![](Link-To-File)
+	  ```
+	-
+	- ## Examples
+	- ### Here is how you could use the Markdown method to embed a picture file found online:
+		- ![](https://logseq.github.io/screenshots/1.png){:height 452, :width 785}
+		- ```markdown
+		  - ![](https://logseq.github.io/screenshots/1.png)
 		  ```
-		  [:audio {:controls true :src "file:C://Users/USERNAME/Folder/audio.ogg"}]
+	-
+	- ### Markdown method -> embed a video file in your assets folder
+		- ```markdown
+		   - ![](../assets/video.mp4)
 		  ```
-		  #+BEGIN_TIP
-		  Make sure to include the `file:` at the start and to use forward slashes `/`
-		  #+END_TIP
-		  #+BEGIN_TIP
-		  You can also access other [HTML attributes for the audio tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio) (see the markup for the `controls` attribute in the above example)
-		  #+END_TIP
-	- ### Embedding an audio file on your PC (using "/Embed HTML")
-		- If you are familiar with HTML markup and [prefer using `<audio>` tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio), you can leverage Logseq's HTML embed feature to preview audio saved on your computer. To do so, type or copy-paste the following line, replacing the source `src` in "quotation marks" with the filepath and file name:
+	-
+	- ### Markdown method -> embed a song saved on your PC:
+		- ![](file:///C:/Users/user/Music/Mix.wav)
+		- ```markdown
+		  - ![](file:///C:\Users\user\Music\Mix.wav)
```

Comment:
You'll probably be interested to know about fragment URIs - https://github.com/logseq/logseq/pull/8300 . These URIs don't scan to the right spot yet because Chromium doesn't support it yet but at least it render and play the whole clip

Reply:
Interesting!!